volume/fs.go

```diff
@@ -30,11 +30,17 @@ func (d *FS) List(ctx context.Context) ([]plugin.Entry, error) {
 	return List(ctx, d, "")
 }
 
-func consumeExec(ctx context.Context, result plugin.ExecResult) (*bytes.Buffer, error) {
+func exec(ctx context.Context, executor plugin.Execable, cmdline []string) (*bytes.Buffer, error) {
+	result, err := executor.Exec(ctx, cmdline[0], cmdline[1:], plugin.ExecOptions{})
```

Comment:
Don't see the cmdline being used anywhere else in the code, including the errors. Is that intentional?

Reply:
Yes, the callers generally add it to their errors.